.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,111 @@
+---
+name: Bug Report
+description: Problems and issues with code or docs
+labels: ["kind:bug"]
+body:
+  - type: dropdown
+    attributes:
+      label: Apache Airlfow Version
+      description: What Apache Airflow version are you using?
+      multiple: false
+      options:
+        - "2.1.3 (latest)"
+        - "2.1.2"
+        - "2.1.1"
+        - "2.1.0"
+        - "2.0.2"
+        - "2.0.1"
+        - "2.0.0"
```

Should we have a "main" option too?
```suggestion
        - "2.0.0"
        - "main"
```